Core/Core/Logging/Analytics.cs

```diff
@@ -221,17 +221,20 @@ public enum Events
 
         if (customProperties != null)
         {
-          properties = properties.Concat(customProperties).ToDictionary(kvp => kvp.Key, kvp => kvp.Value);
+          foreach (KeyValuePair<string, object> customProp in customProperties)
+          {
+            properties[customProp.Key] = customProp.Value;
+          }
         }
 
         string json = JsonConvert.SerializeObject(new { @event = eventName.ToString(), properties });
 
         var query = new StreamContent(new MemoryStream(Encoding.UTF8.GetBytes("data=" + HttpUtility.UrlEncode(json))));
 
-        using HttpClient client = Http.GetHttpProxyClient();
+        using HttpClient client = new();
         client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("text/plain"));
         query.Headers.ContentType = new MediaTypeHeaderValue("application/json");
-        client.PostAsync(MIXPANEL_SERVER + "/track?ip=1", query);
+        var res = await client.PostAsync(MIXPANEL_SERVER + "/track?ip=1", query).ConfigureAwait(false);
```

Comment:
If we aren't using the variable `res`, we should omit it.

Is there any value in calling `EnsureSuccessStatusCode`, it would be nice to know in our logs that there was a failure case, especially if there is no logging performed by the client.